src/tests/ftest/pool/evict_test.py

```diff
@@ -107,13 +111,16 @@ def evict_badparam(self, test_param):
             self.fail("Invalid yaml parameters - check \"params\" values")
         try:
             # call daos evict api directly
```

Comment:
perhaps, dated comment?
It seems that we are now using the dmg command rather than the API

Reply:
Done. Thank you for catching this.